h/static/scripts/directive/annotation.js

```diff
@@ -327,38 +394,17 @@ function AnnotationController(
       return;
     }
 
-    if (model.user) {
+    if (domainModel.user) {
       // User is logged in, save to server.
       // Highlights are always private.
-      model.permissions = permissions.private();
-      model.$create().then(function() {
-        $rootScope.$emit('annotationCreated', model);
+      domainModel.permissions = permissions.private();
+      domainModel.$create().then(function() {
```

Comment:
This _is_ missing a catch but is the same as on master, so I'd like to leave it to a separate pr